runhouse/resources/hardware/cluster.py

```diff
@@ -56,7 +56,8 @@ class Cluster(Resource):
     SERVER_STOP_CMD = f'pkill -f "{SERVER_START_CMD}"'
     # 2>&1 redirects stderr to stdout
     START_SCREEN_CMD = f"screen -dm bash -c \"{SERVER_START_CMD} 2>&1 | tee -a '{SERVER_LOGFILE}' 2>&1\""
-    RAY_START_CMD = f"ray start --head --port {DEFAULT_RAY_PORT}"
+    START_NOHUP_CMD = f"nohup {SERVER_START_CMD} >> {SERVER_LOGFILE} 2>&1 &"
+    RAY_START_CMD = "ray start --head --port 6379"
```

Comment:
can we leave this as `DEFAULT_RAY_PORT`? makes it more clear and also in case this ever gets changed